pkg/endpoint/identifiers.go

```diff
@@ -41,6 +41,23 @@ func (e *Endpoint) GetK8sNamespaceAndPodName() string {
 	return e.K8sNamespace + "/" + e.K8sPodName
 }
 
+// GetK8sCEPResourceName returns the corresponding K8s CiliumEndpoint resource name
+// for this endpoint (without the namespace)
+func (e *Endpoint) GetK8sCEPResourceName() string {
```

Comment:
Not sure I follow. The pod name is set upon initial creation of the endpoint, so this should never be empty if the corresponding endpoint is created for a pod In CNI ADD.

It will always be empty for endpoints without a pod (e.g. host endpoint, health endpoint), and it will always be populated for endpoints backed by a pod, _even_ if we haven't synced the corresponding pod yet.

See also https://github.com/cilium/cilium/commit/e20b16d70843530aefad9ac8c3ae0834c7b6b057

Reply:
Documented!